Rakefile

```diff
@@ -20,3 +20,12 @@ namespace :doc do
   rescue LoadError
   end
 end
+
+desc "SPIKE: Generate the API client files based on route docs."
```

Comment:
This is a small rake task that will read the deployments route definition from octokit/routes. It's not smart. It assumes the octokit/routes are checked out parallel to octikt.rb. It also hard-codes everything and does no error handling.

I assume that a proper rake task would read whatever route files are available and infer what client files should be generated, &c.